general/contact.rst

```diff
@@ -45,12 +45,10 @@ limited, so it is probably **not** the best place to discuss OP-TEE.
 
 Vulnerability reporting
 ***********************
-Please send an email to the address mentioned above (**not** to TEE-dev). Don't
-include any details at this point, just mention that you'd like to report a
-security issue. An engineer from the core OP-TEE team will get back to you for
-further communication and discussions about your findings. Please also read the
-:ref:`disclosure_policy` page and especially the :ref:`reporting_issues`
-section, so you are aware of the rules we are following.
+The OP-TEE project as part of the TrustedFirmare.org organization are using the
```

Comment:
project: are or is?

Reply:
"is" is the correct here I think. I've updated it.